viewing_party/party.py

```diff
@@ -1,23 +1,165 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    movie = {}
+    movie_set = {title, genre, rating}
+
+    if None in movie_set:

```

Comment:
The description said that we should return `None` if any of the inputs were falsy. `None` is _one_ falsy value, but there are others that this would not detect. Consider

```python
    if not all(movie_set):
```

or even the more basic, but explicitly readable
```python
    if not title or not genre or not rating:
```